src/ape_ethereum/transactions.py

```diff
@@ -404,3 +424,13 @@ def _decode_ds_note(self, log: Dict) -> Optional[ContractLog]:
             transaction_hash=log["transactionHash"],
             transaction_index=log["transactionIndex"],
         )
+
+
+class SharedBlobReceipt(Receipt):
+    blob_gas_used: int
+    blob_gas_price: int
+
```

Comment:
can't actually fetch the blob without access to a consensus node, right?

Reply:
I used Alchemy to get the receipt.

Reply:
Oh I mean is the actual blob data fetchable via RPC? AFAIK this type of transaction is just a commitment to a particular blob, which must be fetched from a consensus node

Reply:
i have no idea how to get the blob but the tx.data is the like the proof or something (summarizing discussion from elsewhere)